shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1092,7 +1115,7 @@ \subsection{Chain Transition}
     \left(
       \begin{array}{r@{~\in~}lr}
         \var{s_{now}} & \Slot & \text{current slot} \\
-        \var{pp_n} & \PParams & \text{new protocol parameters} \\
+        \var{gkeys} & \VKeyGen & \text{genesis keys} \\
```

Comment:
an element of the powerset of `VKeyGen` ?

Reply:
yep, it's a set of genesis keys.